### PR TITLE
Update centos_bootstrap.sh

### PR DESCRIPTION
**Pull request by jalbrizio**
added changes to make yeti run on centos.

### CODE REVIEW

**Comment by jalbrizio** [comment]
The build was run on ubuntu. 

Can it be rerun on centos?

**Comment by jalbrizio** [comment]
I am not sure how this fails as nothing changes in the python code. How does the build fail?

**Comment by tomchop** [comment]
Travis seems to be failing on a requests / urllib3 exception

File: extras/centos_bootstrap.sh [hunk 1]
```diff
@@ -21,14 +21,24 @@ yum update -y && yum upgrade -y
 ### Install the YETI Dependencies
 yum groupinstall "Development Tools" -y
 yum install epel-release
-yum install python-pip git mongodb-org python-devel libxml2-devel libxslt-devel zlib-devel redis firewalld yarn -y
+yum install python-pip git mongodb-org python-devel libxml2-devel libxslt-devel zlib-devel redis firewalld yarn vim curl wget net-tools nginx uwsgi -y 
 pip install --upgrade pip
+pip install django libtaxii pyOpenSSL uwsgi cabby django-solo taxii_services python-dateutil
```
Comment (tomchop) [inline extras/centos_bootstrap.sh]: Why are you installing `django`?

Comment (jalbrizio) [reply extras/centos_bootstrap.sh]: good catch it looks like the install requirements from the TAXIIProject yeti project that I was looking at got added here by mistake. I'm cleaning that up now. it should only be uwsgi.

**Comment by jalbrizio** [comment]
I updated the script and removed unneeded pip installs and fixed a typo in the uwsgi service.

**Comment by tomchop** [comment]
Awesome, thanks!

**Pull request merged**